Forms/Fields/DateOfBirth.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace DUCK.Forms.Fields
+{
+	public struct DateOfBirth
+	{
+		public int day;
```

Comment:
Lowercase?

Reply:
Is was following Unity standards for Struct. Ill take a look at our convention and see if Structs are different

Reply:
Why is it a struct ?